.github/resources/pypiserver/base/pypiserver.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pypi-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: pypi-server
+  template:
+    metadata:
+      labels:
+        app: pypi-server
+    spec:
+      containers:
+        - image: "quay.io/harshad16/pypi-server"
+          imagePullPolicy: IfNotPresent
+          name: pypi-server
+          command:
+            - pypi-server
+          args:
+            - run
+            - '--host'
+            - '0.0.0.0'
+            - '--port'
+            - '8080'
+            - '--disable-fallback'
+            - '--verbose'
+            - '/opt/app-root/packages'
+          ports:
+            - containerPort: 8080
+              protocol: TCP
+          resources:
+            requests:
+              cpu: "500m"
+              memory: 1Gi
+            limits:
+              cpu: "500m"
+              memory: 1Gi
```

Comment:
we want to be super conservative here, lower these to `0.02` cpu and `0.5Mi` memory